ivy/data_classes/array/experimental/layers.py

```diff
@@ -1117,3 +1117,57 @@ def rfftn(
             The result of the RFFT operation.
         """
         return ivy.rfftn(self._data, s=s, axes=axes, norm=norm, out=out)
+
+    def sliding_window(
+        self: ivy.Array,
+        window_size: Union[int, Tuple[int, int], Tuple[int, int, int]],
+        stride: Union[int, Tuple[int, int]] = 1,
+        dilation: Union[int, Tuple[int, int]] = 1,
+        padding: Union[str, int, Sequence[Tuple[int, int]]] = "VALID",
+        /,
+        *,
+        data_format="NCHW",
+    ) -> ivy.Array:
```

Comment:
Also, since `stride`, `dilation` and `padding` have default values too, I think the `/, *` should be before them.

Reply:
yes, I'll make the changes accordingly